wgpu-types/src/lib.rs

```diff
@@ -1116,33 +1116,30 @@ pub struct PassChannel<V> {
     pub read_only: bool,
 }
 
-/// Describes a color attachment to a [`RenderPass`].
+/// Describes a color attachment to a render pass.
 #[repr(C)]
 #[derive(Clone, Debug)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
-pub struct RenderPassColorAttachmentDescriptorBase<T> {
-    /// Texture attachment to render to. Must contain [`TextureUsage::OUTPUT_ATTACHMENT`].
+pub struct RenderPassColorAttachmentDescriptor<T, P> {
+    /// The view to use as an attachment.
     pub attachment: T,
-    /// MSAA resolve target. Must contain [`TextureUsage::OUTPUT_ATTACHMENT`]. Must be `None` if
-    /// attachment has 1 sample (does not have MSAA). This is not mandatory for rendering with multisampling,
-    /// you can choose to resolve later or manually.
+    /// The view that will receive the resolved output if multisampling is used.
     pub resolve_target: Option<T>,
-    /// Color channel.
-    pub channel: PassChannel<Color>,
+    /// What operations will be performed on this color attachment.
+    pub channel: P,
```

Comment:
This field is called `channel` here, because it used to be a `PassChannel`. In the equivalent `wgpu-rs` structure, this field is called `ops` because it is of the type `Operations`.